scripts/run-depcheck.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+set -ex
+
+# Create a temporary depcheckrc file for 'jaeger-ui'
+tempfile_jaeger=$(mktemp /tmp/depcheckrc.XXXXXX.json)
+node scripts/generateDepcheckrcJaegerUI.js "$tempfile_jaeger"
```

Comment:
you could've done this more "unixy" by writing to stdout from the Node script and redirecting here to the temp file. This way there would be fewer dependencies between the steps, the Node script is easier to debug, and it would be simpler (no writing to files).

Reply:
I was earlier thinking of the same to redirect the output using `>` operator in Unix.
Would create a sister PR to rectify the same.